src/doctrines/Monoidal.jl

```diff
@@ -78,7 +75,8 @@ show_latex(io::IO, expr::ObExpr{:munit}; kw...) = print(io, "I")
 The signature (but not the axioms) is the same as a braided monoidal category.
 """
 @signature MonoidalCategory(Ob,Hom) => SymmetricMonoidalCategory(Ob,Hom) begin
-  braid(A::Ob, B::Ob)::Hom(otimes(A,B),otimes(B,A))
+  braid(A::Ob, B::Ob)::((A ⊗ B) → (B ⊗ A))
+  @op braid :σ
```

Comment:
We need to export the Unicode operator for braiding. It would also be nice to add a unit test.

Reply:
Fixed in commit 560bb3e